actions/branch-out-upload/action.yml

```diff
@@ -47,21 +47,25 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Install JUnit Enhancer and Run
+    - name: Install JUnit Enhancer
       shell: bash
       env:
         VERSION: ${{ inputs.junit-enhancer-version }}
-        JUNIT_PATH: ${{ inputs.junit-file-path }}
       run: |
         echo "::group::Install JUnit Enhancer"
-        go install github.com/smartcontractkit/quarantine/cmd/junit-enhancer@${VERSION}
+        GOPROXY=direct go install github.com/smartcontractkit/quarantine/cmd/junit-enhancer@${VERSION}
```

Comment:
Is it a common issue to get a stale version? If we do this, we have a higher chance of being limited by GitHub.